pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -202,11 +199,19 @@ protected void createJwtProfile(final TokenCredentials credentials, final JWT jw
         if (subject == null) {
             throw new TechnicalException("JWT must contain a subject ('sub' claim)");
         }
-
         if (!subject.contains(CommonProfile.SEPARATOR)) {
             subject = JwtProfile.class.getName() + CommonProfile.SEPARATOR + subject;
         }
 
+        final Date expirationTime = claimSet.getExpirationTime();
+        if (expirationTime != null) {
+            final Date now = new Date();
+            if (expirationTime.before(now)) {
+                logger.error("The JWT is expired: no profile is built");
+                return;
```

Comment:
I will merge it as is and we'll revisit that for the 2.0.0 version if necessary.